recipe/meta.yaml

```diff
@@ -42,8 +42,10 @@ requirements:
     - spdlog 1.10 *_0
     - fmt
     - yaml-cpp-static              # [unix]
-    - libcurl >=7.86               # [unix]
-    - libcurl-static >=7.86        # [unix]
+    - libcurl >=7.87               # [linux]
+    - libcurl-static >=7.87        # [linux]
+    - libcurl 7.86                 # [osx]
+    - libcurl-static 7.86          # [osx]
```

Comment:
Why?

Reply:
For an unknown reason, upgrading to curl 7.87 breaks micromamba on OSX, it cannot download anything.

Reply:
There is an SSL error (ReadSSL() returns -50). 

Hopefully this fixes it: https://github.com/curl/curl/pull/10235